.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: File a bug report
+title: '[Bug] '
+labels: ["c.Bug 🐛", "s.ToInvestigate"]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
```

Comment:
Maybe allow linking of other similar issues here? In case the issue is different but very close to another issue.

Reply:
I think if there is a related issue, the person can put it in bug description. There's no way to provide a "specify: " within this section. Are you suggesting creating another field for linking related issues?

Reply:
Yeah, perhaps there can be a field for linking related issues, since that is a fairly common occurrence.